src/test/java/net/datafaker/providers/base/TimeTest.java

```diff
@@ -75,6 +83,16 @@ void testBetweenThenLargerThanNow() {
             .hasMessage("Invalid time range, the upper bound time is before the lower bound.");
     }
 
+    @Test
+    void testBetweenWithPattern() {
```

Comment:
what's difference between this test and existing `testBetween`?

Reply:
This test for `between` with pattern

Reply:
There are 2 tests below testing this for future and past
It would make sense to similar test naming like `betweenWithMask`
and similar style for code of the test. It would simplify reading and understanding for others in future

Reply:
Fixed